pkg/report/spdx/spdx.go

```diff
@@ -0,0 +1,152 @@
+package spdx
+
+import (
+	"io"
+	"time"
+
+	"golang.org/x/xerrors"
+
+	"github.com/google/uuid"
+	"github.com/spdx/tools-golang/jsonsaver"
+	"github.com/spdx/tools-golang/spdx"
+	"github.com/spdx/tools-golang/tvsaver"
+	"k8s.io/utils/clock"
+
+	ftypes "github.com/aquasecurity/fanal/types"
+	"github.com/aquasecurity/trivy/pkg/types"
+)
+
+const (
+	SPDXVersion         = "SPDX-2.1"
```

Comment:
```suggestion
	SPDXVersion         = "SPDX-2.2"
```

The latest version of the [SPDX specification](https://spdx.github.io/spdx-spec/) is 2.2 :smiley: